test/Fondue.test.js

```diff
@@ -322,7 +322,11 @@ describe("Layout features", () => {
 
 		expect(fondue.featureChars["DFLT"]["dflt"]).toEqual(
 			expect.objectContaining({
-				onum: ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"],
+				onum: {
+					type: 1,
+					input: ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"],
+					alternateCount: [],
```

Comment:
Does this actually test the code that was added?

For documentation purposes it would be nice if the tests would use the same terminology as the implementation. So having tests for lookupType 1,2,3,4 etc.

Reply:
I expanded the tests to account for all 3 lookups we currently provide for in Wakamai Fondue. Please see latest few commits :-)

Reply:
Nice! 👍